data-structures/stacks-1.js

```diff
@@ -0,0 +1,16 @@
+export function getTop(stack) {
+  return stack.peek();
+}
+
+export function addToTop(stack, value) {
+  return stack.push(value);
+}
+
+export function takeTop(stack) {
+  return stack.pop();
+}
+
+export function isEmpty(stack) {
+  if (stack.peek() === undefined) return true;
+  return false;
```

Comment:
Same here. In general, if you find yourself writing `if (foo) return true; else return false;` then you should consider replacing it with `return foo;`.